warning: `QListWidget::setItemSelected()` is deprecated

```shell
/home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/ListEditor.cpp: In member function ‘void OpenMS::Internal::ListTable::createNewRow()’:
/home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/ListEditor.cpp:334:22: warning: ‘void QListWidget::setItemSelected(const QListWidgetItem*, bool)’ is deprecated: Use QListWidgetItem::setSelected() instead [-Wdeprecated-declarations]
  334 |       setItemSelected(item, true);
      |       ~~~~~~~~~~~~~~~^~~~~~~~~~~~
In file included from /usr/include/qt/QtWidgets/QListWidget:1,
                 from /home/USER/Development/OpenMs/src/openms_gui/include/OpenMS/VISUAL/ListEditor.h:45,
                 from /home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/ListEditor.cpp:35:
/usr/include/qt/QtWidgets/qlistwidget.h:263:10: note: declared here
  263 |     void setItemSelected(const QListWidgetItem *item, bool select);
      |          ^~~~~~~~~~~~~~~
```

--- a/src/openms_gui/source/VISUAL/ListEditor.cpp
+++ b/src/openms_gui/source/VISUAL/ListEditor.cpp
@@ -331,7 +331,7 @@ namespace OpenMS
       item->setTextAlignment(Qt::AlignLeft | Qt::AlignVCenter);
       item->setFlags(Qt::ItemIsSelectable | Qt::ItemIsEnabled | Qt::ItemIsEditable);
       addItem(item);
-      setItemSelected(item, true);
+      item->setSelected(true);
       setCurrentRow(row(item));
       itemActivated(item);
       edit(currentIndex());

